base/compiler/abstractinterpretation.jl

```diff
@@ -58,6 +61,10 @@ function abstract_call_gf_by_type(interp::AbstractInterpreter, @nospecialize(f),
             end
             push!(infos, MethodMatchInfo(matches))
             append!(applicable, matches)
+            for _ in 1:length(matches)
```

Comment:
I think for code readability, these ops are clearer if they go in the same loop
```suggestion
            for m in matches
                push!(applicable, m)
```

Reply:
I agree. Applied.